src/main/java/ua/mateacademy/tania/patterns/abstractfactory/AbstractFactoryApp.java

```diff
@@ -0,0 +1,15 @@
+package ua.mateacademy.tania.patterns.abstractfactory;
+
+import ua.mateacademy.tania.patterns.abstractfactory.polish.PolishFactory;
+
+public class AbstractFactoryApp {
+    public static void main(String[] args) {
+        Factory factory = new PolishFactory();
```

Comment:
You use concretic factory|class. It is hardcode. Read https://www.baeldung.com/java-abstract-factory-pattern